src/core/webidl.js

```diff
@@ -141,36 +137,44 @@ function registerHelpers() {
     }
     return a.outerHTML;
   });
+  hb.registerHelper("trivia", writeTrivia);
```

Comment:
I can't tell exactly here, but it looks like you are registering the "trivia" helper  inside "tryLink"? Is that correct? or should it be outside like the rest?